app/controllers/admin/settings/projects_settings_controller.rb

```diff
@@ -45,14 +45,17 @@ def validate_enabled_modules
 
       module_missing_deps = OpenProject::AccessControl
         .modules
-        .detect { |m| m[:dependencies] && enabled_modules.include?(m[:name]) && (m[:dependencies] & enabled_modules) != m[:dependencies] }
+        .select do|m| m[:dependencies] && enabled_modules.include?(m[:name]) && (m[:dependencies] & enabled_modules) != m[:dependencies] }
```

Comment:
```suggestion
        .select { |m| m[:dependencies] && enabled_modules.include?(m[:name]) && (m[:dependencies] & enabled_modules) != m[:dependencies] }
```

Reply:
That was fast @aaron-contreras !⚡️  fixed it :)

Reply:
I didn't even know or couldn't remember the `list_of_messages` helper. Thanks for fixing and reviewing so quickly both of you 🙏